src/client/AxiosClient.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { Configuration } from "../utils/Configuration";
+
+/**
+ * Client to generate instance of Axios that is already configured to call required url
```

Comment:
Is the client setup needed?

Reply:
I discussed this with Mel before Christmas, I had 3 solutions ready to go.
1) Leave logic where it was for both Secret retrieval logic and Axios Instance setup in the service
2) Leave logic for Axios Instance setup where it was and move Secret retrieval logic into Configuration
3) Move both into the respective locations you see on this PR

We agreed on 3.

I think it is cleaner, removes a bit of unnecessary complexity from the lookupPostcode method and promotes modularity.

But if you have any concerns or any reasons why you think this is bad, then would be happy to discuss.

Reply:
It's giving me Java factory pattern flashbacks!!

Reply:
Not a bad thing!